docs/getting-started.md

```diff
@@ -0,0 +1,67 @@
+# Getting Started
+
+## Installation
+
+### Building from source
+
+Prerequisites: The [Go](https://golang.org/doc/install) compiler **version 1.14** (we use
+go modules) or later and GNU make. The go bin path (usually `$HOME/go/bin`) should be
+on your system `PATH`.
+
+Run `make install` to build the code. The `regen` binary serves as both the blockchain daemon, as well as
+the command line blockchain client.
+
+## Configuring the command line client
+
+By default the command line client will connect to a [local node](#running-a-local-node).
+To connect to a testnet or devnet, you must know the network's chain ID and the address of the node. More information on the current testnets and devnet can be found in the [testnets repository](https://github.com/regen-network/testnets).
+
+`regen` can be configured to connect to a testnet node automatically by setting these
+parameters in `$HOME/.regen/config/config.toml`. This file can be generated
+automatically by running:
+
+```sh
+regen init --chain-id [chain ID] --node [moniker]
+```
+If you're wanting to connect to our devnet that launched during the [Open Climate Collabathon](https://www.collabathon.openclimate.earth/), the chain-id is `regen-devnet-1`.
+
+
+If not connecting to a live testnet or devnet, initialize the node config for local development:
+
+```sh
+regen init [moniker]
+```
+
+
+## Running a node
```

Comment:
Between `init` and `start` I believe there are still some other steps to run. We could take inspiration from https://docs.cosmos.network/master/run-node/run-node.html

Reply:
Yes, a followup pr?